[2024-12-09T20:51:44.745+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-09T20:51:44.763+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_data_pipeline_mongodb_complex.load_data manual__2024-12-09T20:45:56.898218+00:00 [queued]>
[2024-12-09T20:51:44.772+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_data_pipeline_mongodb_complex.load_data manual__2024-12-09T20:45:56.898218+00:00 [queued]>
[2024-12-09T20:51:44.773+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 2
[2024-12-09T20:51:44.785+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_data> on 2024-12-09 20:45:56.898218+00:00
[2024-12-09T20:51:44.792+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1003) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-09T20:51:44.794+0000] {standard_task_runner.py:72} INFO - Started process 1005 to run task
[2024-12-09T20:51:44.794+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_data_pipeline_mongodb_complex', 'load_data', 'manual__2024-12-09T20:45:56.898218+00:00', '--job-id', '335', '--raw', '--subdir', 'DAGS_FOLDER/etl_data_pipeline_mongodb_complex.py', '--cfg-path', '/tmp/tmp6j3si8ye']
[2024-12-09T20:51:44.795+0000] {standard_task_runner.py:105} INFO - Job 335: Subtask load_data
[2024-12-09T20:51:45.043+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_data_pipeline_mongodb_complex.load_data manual__2024-12-09T20:45:56.898218+00:00 [running]> on host 8553af9ff67b
[2024-12-09T20:51:45.102+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_data_pipeline_mongodb_complex' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-09T20:45:56.898218+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-09T20:45:56.898218+00:00'
[2024-12-09T20:51:45.103+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-09T20:51:45.104+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-09T20:51:45.104+0000] {logging_mixin.py:190} INFO - Current task name:load_data state:running start_date:2024-12-09 20:51:44.764503+00:00
[2024-12-09T20:51:45.105+0000] {logging_mixin.py:190} INFO - Dag name:etl_data_pipeline_mongodb_complex and current dag run status:running
[2024-12-09T20:51:45.105+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-09T20:51:45.142+0000] {logging_mixin.py:190} INFO - Données chargées dans MongoDB.
[2024-12-09T20:51:45.144+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-09T20:51:45.154+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-09T20:51:45.155+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_data_pipeline_mongodb_complex, task_id=load_data, run_id=manual__2024-12-09T20:45:56.898218+00:00, execution_date=20241209T204556, start_date=20241209T205144, end_date=20241209T205145
[2024-12-09T20:51:45.163+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-12-09T20:51:45.163+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-12-09T20:51:45.164+0000] {logging_mixin.py:190} INFO - Dag name:etl_data_pipeline_mongodb_complex queued_at:2024-12-09 20:45:56.905278+00:00
[2024-12-09T20:51:45.164+0000] {logging_mixin.py:190} INFO - Task hostname:8553af9ff67b operator:PythonOperator
[2024-12-09T20:51:45.210+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-09T20:51:45.355+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-09T20:51:45.358+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
